rediscluster/pipeline.py

```diff
@@ -262,12 +276,18 @@ def load_scripts(self):
     def watch(self, *names):
         """
         """
-        raise RedisClusterException("method watch() is not implemented")
+        if not names:
+            raise RedisClusterException("WATCH command needs at least on name in this pipeline")
+
+        return self.execute_command('WATCH', *names)
 
-    def unwatch(self):
+    def unwatch(self, *names):
         """
         """
-        raise RedisClusterException("method unwatch() is not implemented")
+        if not names:
+            raise RedisClusterException("UNWATCH command needs at least on name in this pipeline")
```

Comment:
Possibly change the error message to `UNWATCH command needs at least one name when used in a pipeline`